L1Trigger/L1TTrackMatch/plugins/L1TrackVertexAssociationProducer.cc

```diff
@@ -148,6 +148,14 @@ class L1TrackVertexAssociationProducer : public edm::global::EDProducer<> {
     std::vector<double> deltaZMaxEtaBounds_;
     std::vector<double> deltaZMax_;
   };
+  struct TTTrackLinkLimitSelector {
+    TTTrackLinkLimitSelector() {}
```

Comment:
```suggestion
    TTTrackLinkLimitSelector() = default;
```
or just leave it out